code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -115,11 +115,15 @@
 	required_temp = 550 + rand(-49,49)
 
 
-/datum/chemical_reaction/reagent_explosion/penthrite_explosion
-	required_reagents = list(/datum/reagent/medicine/C2/penthrite = 1, /datum/reagent/phenol = 1, /datum/reagent/acetone_oxide = 1)
-	required_temp = 315
+/datum/chemical_reaction/reagent_explosion/penthrite_explosion_epinephrine
+	required_reagents = list(/datum/reagent/medicine/C2/penthrite = 1, /datum/reagent/medicine/epinephrine = 1)
```

Comment:
meth explosion 2, electric boogaloo

Reply:
except this is even stronger, because it can react in your body

make two smoke grenades that are synced up to the same signaler signal. put epinephrine in one grenade and penthrite in the other. trigger them both at once, then watch as everyone nearby you who doesn't have their internals on instantly explodes

or just stick penthrite in a hypospray, then click someone once with a penthrite hypospray and once with an epinephrine pen to cause them to explode

Reply:
you discovered rdx + teslium incredibly late

Reply:
except you need to use a hypospray for teslium, because there's no teslium medipens